bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/GCData.java

```diff
@@ -47,6 +48,8 @@ public final class GCData {
 	public float[] lineDashes;
 	public float lineMiterLimit = 10;
 	public int alpha = 0xFF;
+	public int deviceZoom;
+	public int nativeDeviceZoom;
```

Comment:
Letting these 2 be public can be dangerous since they can be set separately, which contradicts what the new method `setNativeDeviceZoom` does.
Make them private and provide getters instead